src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -192,7 +192,7 @@ class Microsoft::Terminal::Core::Terminal final :
 #pragma region TextSelection
     // These methods are defined in TerminalSelection.cpp
     std::vector<SMALL_RECT> _GetSelectionRects() const;
-    const SHORT _ExpandWideGlyphSelectionLeft(const SHORT xPos, const SHORT yPos) const noexcept;

```

Comment:
Wait why do these have to be _not_ `noexcept`? (I think I'm missing it)

Reply:
That's what I thought. BUT, `const auto attr = _buffer->GetCellDataAt(position)->DbcsAttr();` (speciffically the `_buffer->GetCellDataAt(position)` part) isn't always safe to use. If you're outside of the buffer, you crash.

The exact same thing happened with copying after a resize event. You'd point to a place that doesn't exist.